src/raster/r.mblend/testsuite/test_rmblend.py

```diff
@@ -0,0 +1,84 @@
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+from grass.gunittest.gmodules import SimpleModule
+
+
+class TestRMblend(TestCase):
+
+    low_res = "low_res"
+    high_res_1 = "high_res_1"
+    high_res_2 = "high_res_2"
+    result_1 = "result_1"
+    result_2 = "result_2"
+
+    @classmethod
+    def setUpClass(cls):
+
```

Comment:
The empty line is not needed here, but if you run Pylint it will make you add a comment in this place.

Reply:
I added a doc string to the class but left the methods without it, like in other unit tests. Since these tests are so simple the doc string won't be adding much.

The space between class/method declarations and body is a matter of style. Makes it easier for me read. Pylint is fine with it.